pkg/controller/memberstatus/memberstatus_controller.go

```diff
@@ -41,6 +42,7 @@ const (
 	hostConnectionTag statusComponentTag = "hostConnection"
 	resourceUsageTag  statusComponentTag = "resourceUsage"
 	routesTag         statusComponentTag = "routes"
+	cheIntegrationTag statusComponentTag = "cheIntegration"
```

Comment:
Shouldn't it be `che` for the sake of consistency? When the component tag matches the component name?

Reply:
I thought just `che` could be a bit too ambiguous but yeah maybe it's better for consistency and having the tag match the component name. Done in cfd576103ff935a95eeb0bf2b47a24bfebb4c7a6